selenium/src/test/java/au/org/emii/portal/tests/BrowserStackTest.java

```diff
@@ -0,0 +1,19 @@
+package au.org.emii.portal.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class BrowserStackTest extends BaseTest {
+
+    @Test(invocationCount = invocationCount, threadPoolSize = threadPoolSize)
+    public void BrowserStackTest() {
```

Comment:
Tests should normally be methods not constructors e.g. testBrowserStackSearch.  Using a constructor makes it difficult to use many of the other features of testng such as @Before and @After which all assume you are writing tests as methods.   Also, I wouldn't normally put a doing operation like this into a constructor - it would normally be used for initialising the instance/setting instance variables.

This seems to be testing that browserstack integration is working.  I don't think we need to test that explicity - we'll find that out when we try to perform other tests.

Would also not put a println statement in a test method like that except during local debugging.